docs/runningLocally.md

```diff
@@ -24,7 +24,7 @@ after that the app should track all the changes/editis in any file you share
 
 ## preforming simple manual test
 
-you will need redis running locally 
+you will need redis running locally. 
```

Comment:
```suggestion
you will need redis running locally using the command `redis-server`. 
```
If possible, it would be good to start redis as part of the main run command. I'm not sure if it's as straightforward as just including it in the `dev` command?

Reply:
yeah would be good to have it in the script. I'll add it and check that it works

Reply:
My knowledge of redis is almost non-existent sadly, so there may well be a reason it's been left out of the existing scripts

Reply:
Adding in the first script seemed to work fine.